pnpm-lock.yaml

```diff
@@ -368,22 +368,26 @@ importers:
       '@next/eslint-plugin-next': 12.3.1-canary.1
```

Comment:
This lockfile change seems unnecessary can we revert it?

Reply:
Reverted